### DISCUSSION TITLE
小米运动 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#70] [模板讨论]
小米运动 模板问题反馈评论区
有问题可以下面反馈。

**Comment by OwnerCM** [comment]
不支持最新邮箱注册账户

**Reply by wjf0214** [reply]
邮箱的之前做好了，还没有更新。如果手机号版本的不需要的话，我就直接替换了。需要的话，我就再新增一个版本。

**Reply by bombjinne** [reply]
用的邮箱版提示成功了但是微信没有同步，求解

**Reply by wjf0214** [reply]
成功了的话，那就没问题。同步是小米运动和微信的事情。数据已经到小米运动了。之前有人提出的方案是注销小米账号，再申请注册重试。我现在不清楚还有没有效果。。没同步的话，也没什么好的处理方式。

**Comment by leic4u** [comment]
连续三天报这个错了。感觉好像门被关了😂

![image](https://github.com/wjf0214/qd-templates/assets/32786903/dd618e55-672e-4b08-a87b-4ca3481dd69c)

**Reply by bombjinne** [reply]
我用邮箱没问题，都是新注册的账号

**Reply by wjf0214** [reply]
嗯，用邮箱吧。早期是手机号，后面新注册只有邮箱了。现在应该全面过渡到邮箱，估计手机api关闭了吧，而且账号不是小米账号的。

**Reply by leic4u** [reply]
今天试了下又可以运行成功了。

**Comment by Misuzu7** [comment]
2024-07-30 
有人失效没，我的显示成功但是zepp里没有步数

**Comment by hecarli555** [comment]
请问现在是不是要有手环绑定app后，步数才能同步到微信上？
